source/content/guides/decoupled/wp-backend-starters/03-caching.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Decoupled WordPress Backend Starter for Front-End Sites
```

Comment:
https://pr-8713-documentation.appa.pantheon.site/guides/decoupled/wp-backend-starters/manage-settings

@CobyPear this still looks inconsistent. all the other files in this folder have `title: WordPress Backend Starter for Front-End Sites`